docs/getting-started/creating-your-first-bot.mdx

```diff
@@ -191,7 +191,7 @@ event that is automatically called when the bot is ready to use.
 
 ```py
 @bot.slash_command(name = "hello", description = "Say hello to the bot")
-async def say_hello(ctx):
+async def say_hello(ctx: discord.ApplicationContext):
```

Comment:
```suggestion
async def hello(ctx: discord.ApplicationContext):
```

Not part of this PR, but this should be consistent with the actual code example lol.